master/internal/api_master.go

```diff
@@ -22,13 +22,24 @@ import (
 func (a *apiServer) GetMaster(
 	_ context.Context, _ *apiv1.GetMasterRequest) (*apiv1.GetMasterResponse, error) {
 	return &apiv1.GetMasterResponse{
-		Version:     a.m.Version,
-		MasterId:    a.m.MasterID,
-		ClusterId:   a.m.ClusterID,
-		ClusterName: a.m.config.ClusterName,
+		Version:          a.m.Version,
```

Comment:
nit: this is why I don't like formatting so that the values align.

Reply:
what's this about?

Reply:
having other unrelated lines change because you added or removed the longest line in a struct

Reply:
ah yeah agree. someone needs to write a gofmt aware diffing algorithm ha.